python/core/qgsvectorlayereditbuffer.sip

```diff
@@ -148,4 +148,16 @@ class QgsVectorLayerEditBuffer : QObject
     void updateAttributeMapIndex( QgsAttributeMap& attrs, int index, int offset ) const;
 
     void updateLayerFields();
+
+    /** Apply geometry modification basing on provider geometry type.
```

Comment:
If you could add here a hint on what kind of modification can be expected from this method this would help a lot for a reader of the API. (I.e. some reference to Z, M, Multi, others?)

Reply:
+1

Reply:
added more doc to the underling call that do the work: convertToProviderType